LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -77,6 +77,7 @@ public struct CorrectionRangeOverridesEditor: View {
             }
         )
         .alert(isPresented: $showingConfirmationAlert, content: confirmationAlert)
+        .navigationBarTitle("", displayMode: .inline)
```

Comment:
? Is this needed?

Reply:
Yes, this resolves the issue with the empty space between the navigation bar and the "fake title"

Reply:
Nice.  Such magic!  🧙‍♀️